installer/conf/container.conf

```diff
@@ -7,24 +7,18 @@
 	bind 127.0.0.1
 </source>
 
-# Container inventory
 <source>
-	type omi
-	run_interval 60s
-	tag oms.container.containerinventory
-	items [
-		["root/cimv2","Container_ContainerInventory"]
-	]
+        type oms_omi
+        object_name "Container"
+        interval 30s
 </source>
 
-# Image inventory
+# Container inventory
 <source>
-	type omi
-	run_interval 60s
-	tag oms.container.imageinventory
-	items [
-		["root/cimv2","Container_ImageInventory"]
-	]
+  type containerinventory
+  tag oms.containerinsights.ContainerInventory
```

Comment:
nit: everything else is following all lowercase convention. change to containerinventory?